src/code/fault.c

```diff
@@ -115,10 +114,9 @@ u32 Fault_ProcessClient(u32 callback, u32 param0, u32 param1) {
 // minor ordering differences
 void Fault_AddClient(FaultClient* client, void* callback, void* param0, void* param1) {
     OSIntMask mask;
-    u32 alreadyExists;
+    bool alreadyExists = false;
```

Comment:
We don't use bool types because booleans are in many different types since they didn't use it consistently either.
```suggestion
    u32 alreadyExists = false;
```